R/boxr_misc.R

```diff
@@ -27,7 +27,7 @@ box_ls <- function(dir_id = box_getwd(), limit = 100, max = Inf, fields = NULL)
     paste(url_root, "folders", box_id(dir_id), "items", sep = "/")
   )
 
-  filelds_all <- 
+  fields_all <- 
     c("modified_at" ,"content_modified_at", "name", "id", "type",
```

Comment:
I think you also need an entry at line 6 to document the `fields` argument, something like:

```r
#` @param fields Default (`NULL`) returns all possible columns: 
#`  `"modified_at"`, `"content_modified_at"`, `"name"`, `"id"`, `"type"`, `"sha1"`,
#`  `"size"`, `"owned_by"`, `"path_collection"`, `"description"`. 
#`  To return a subset of columns, specify as a character vector.
#` 
```
If you document() again (and let me know that it works as you expect), I think we're there!

Thanks!

Reply:
Oh right! Will do